portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -56,11 +60,31 @@ public void init() {
 
       Double riceConsump = riceStr.equals(UNDEFINED_DATA) ? 0.0 : Double.valueOf(riceStr);
 
-      countryMap.put(country, riceConsump);
+      if (countryNameMap.containsKey(country)) {
+        countryMap.put(countryNameMap.get(country), riceConsump);
+      } else {
+        countryMap.put(country, riceConsump);
+      }
     }
     scanner.close();
   }
 
+  private void fillCountryNameMap() {
+    countryNameMap.put("Russian Federation", "Russia");
+    countryNameMap.put("United States of America", "United States");
+    countryNameMap.put("Viet Nam", "Vietnam");
+    countryNameMap.put("Lao People's Democratic Republic", "Laos");
+    countryNameMap.put("Iran (Islamic Republic of)", "Iran");
+    countryNameMap.put("Sudan (former)", "Sudan");
+    countryNameMap.put("Congo", "DRC");
+    countryNameMap.put("United Republic of Tanzania", "Tanzania");
+    countryNameMap.put("Democratic People's Republic of Korea", "North Korea");
+    countryNameMap.put("Republic of Korea", "South Korea");
+    countryNameMap.put("Venezuela (Bolivarian Republic of)", "Venezuela");
+    countryNameMap.put("Bolivia (Plurinational State of)", "Bolivia");
+    countryNameMap.put("Syrian Arab Republic", "Syria");
```

Comment:
Instead of storing this data by specifying it in the java file, it might make more sense to create a new CSV to contain this relationship.  Then you can import the CSV like you do for the other file.